charts/cryostat/templates/db_deployment.yaml

```diff
@@ -85,12 +85,12 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       volumes:
-      {{- if ((.Values.pvc).enabled) }}
+      {{- if ((.Values.db.resources.pvc).enabled) }}
```

Comment:
```suggestion
      {{- if (.Values.db.pvc).enabled }}
```

I think `.db.resources` refers to container resource requirements :D

Reply:
Ah right, thanks. I initially nested it under `resources`, but then it would be annoying to set PVC settings and CPU/memory separately so I moved it back up a level. I'll change this when I'm back at my computer (probably Monday).

Reply:
> I'll change this when I'm back at my computer (probably Monday).

Sounds good! Happy Black Friday weekends^^